### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by PlainOlSoapBar**
Cleaned up the Dev Environment instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,50 +54,47 @@ Project is designed to utilize django's powerful features.
...
+
+**MYSQL SETUP** (If you are on a Mac, consider switching to Windows if you have one)
+1. Download MySQL over [here](https://dev.mysql.com/downloads/installer/). Download the larger file.
+2. Do the setup and do the full download (spam next until you see the password portion).
+3. Make a password for the root account or a custom username.
+   - Make sure you remember your passwords and username (username is by default root).
+4. Write the username and password in your .env file.
+5. Spam next until you finish and then you’re done.
+
+Next we're onto Django.
```
Comment (misslame) [inline README.md:84]: This isn't Django not sure why it was originally stated as such. The steps below are still MySQL set up just integrating it to the environment. The first few steps above are related to Django. ex. pipenv install should install django

File: README.md [hunk 2]
```diff
@@ -54,50 +54,47 @@ Project is designed to utilize django's powerful features.
...
+4. You should be prompted into the MySQL shell, where you will then create a database by using `CREATE DATABASE mesh;`.
+5. Confirm your database is created with `SHOW DATABASES;`.
+6. Make sure your pip environment is opened with `pipenv shell`.
+7. Type in `python manage.py makemigrations`.
+    - You will most likely be given warnings. Warnings are fine errors aren’t. Google your errors.
+8. Type in `python manage.py migrate`.
+    - Similar to the previous step warnings are fine, errors aren’t.
+9. Make sure your frontend is set up with `npm run build`, then run `python manage.py runserver`, and you should be good to go.
 
 #### Tips:
```
Comment (misslame) [inline README.md:99]: Do you want to add the scripts from https://github.com/LetsMesh/Site/blob/PlainOlSoapBar-patch-1/package.json  ?

File: README.md [hunk 3]
```diff
@@ -54,50 +54,47 @@ Project is designed to utilize django's powerful features.
...
-4. Initialize the virtual environment for yourself and installs virtual environment dependencies -> `pipenv install`
-**IMPORTANT** Get the .env file from someone and store it on your root directory '/SITE' if you haven't already before running this command.
-5. Activate the virtual environment -> `pipenv shell`
-6. Go to `Site/meshapp/` and run an `npm install`
-7. Build the frontend -> in the same directory as step 7 run `npm run build`
-8. Return to the root directory and start the backend server / run the application -> `python manage.py runserver`
-MYSQL SETUP (windows, if you have a mac, consider switching to a windows)
-1. Download Mysql over here https://dev.mysql.com/downloads/installer/ ( i downloaded the file that was 331.3 M)
-2. Do the setup and do the full download (spam next until you see the password portion) 
-3. Make sure you are not using the legacy version, select the 8.x version
```
Comment (misslame) [inline README.md:81]: why was this removed?

**Review by misslame — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (PlainOlSoapBar) [reply README.md:81]: This wasn't removed. I reformatted it.

File: README.md (hunk 2 above)
Comment (PlainOlSoapBar) [reply README.md:99]: Sure, what command do you use to run these scripts?

Comment (misslame) [reply README.md:99]: Should be  npm run [command]

File: README.md (hunk 3 above)
Comment (misslame) [reply README.md:81]: Maybe im blind where is the "make sure you are not using legacy version"

Comment (PlainOlSoapBar) [reply README.md:81]: You're right I did remove it, I'll add it back in

**Review by chrispybokchoy — APPROVED** [review]
for the .env you can add that it can be found pinned in the current-sprint text channel.
